### PR TITLE
Fix old informations in readme

### PR DESCRIPTION
**Pull request by ghost**
Thank you for this great oss.

```vim
let g:fzf_preview_git_status_command = "git status --short --untracked-files=all | awk '{if (substr($0,2,1) !~ / /) print $2}'"
```

Above is not default anymore. 
To make matters worse, it causes an error and I can't run command if I set this in vimrc.
It is because it has no group.

https://github.com/yuki-ycino/fzf-preview.vim/blob/9719e67fdcd9dc8f938d4398a22ec5f4aca1d875/src/fzf/resource/git-status.ts#L42-L44

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,10 +196,10 @@ and
 :FzfPreviewCommandPalette                         " Execute and edit command history
 :CocCommand fzf-preview.CommandPalette
 
-:FzfPreviewGitActions                         " Interactive git integration. (Required [Fugitive](https://github.com/tpope/vim-fugitive) and [Gina](https://github.com/lambdalisue/gina.vim))
+:FzfPreviewGitActions                         " Interactive git integration. (Required [Fugitive](https://github.com/tpope/vim-fugitive) or [Gina](https://github.com/lambdalisue/gina.vim))
```
Comment (ghost) [inline README.md:199]: this should be `or`, I get confused at first time.

The other sentences are written `or` as well.

https://github.com/yuki-ycino/fzf-preview.vim/blob/9719e67fdcd9dc8f938d4398a22ec5f4aca1d875/README.md#L30

File: README.md [hunk 2]
```diff
@@ -196,10 +196,10 @@ and
...
 :CocCommand fzf-preview.GitActions
 
-:FzfPreviewGitStatus                          " Select git status listed file. (Required [Fugitive](https://github.com/tpope/vim-fugitive) and [Gina](https://github.com/lambdalisue/gina.vim))
+:FzfPreviewGitStatus                          " Select git status listed file. (Required [Fugitive](https://github.com/tpope/vim-fugitive) or [Gina](https://github.com/lambdalisue/gina.vim))
```
Comment (ghost) [inline README.md:202]: ditto

**Comment by yuki-yano** [comment]
Thank you.
Please correct the doc as well.

**Comment by ghost** [comment]
@yuki-ycino 
Here I fix
20a4de8

**Review by yuki-yano — APPROVED** [review]
LGTM!

**Pull request merged**